content/_data/changelogs/weekly.yml

```diff
@@ -9391,6 +9391,94 @@
   # pull: 5029 (PR title: Queue (mainly) style consistency)
   # pull: 5033 (PR title: JDK 11.0.9 / 8.0.272+ compatible FunctionsTest#printThrowable())
 
+- version: '2.266'
+  date: 2020-11-10
+  changes:
+  - type: major bug
+    category: major bug
+    pull: 5046
+    issue: 64040
+    authors:
+    - timja
+    message: |-
+      Fix hidden page elements from radio blocks appearing when they should not.
+  - type: rfe
```

Comment:
FTR https://github.com/jenkinsci/jenkins/pull/5008#issuecomment-723894717, a type might be changed

Reply:
Listed as a bug in e2f776ec83 .  Thanks for catching that!